bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/NMSSM_XYH_ggtt/Template/proc_card.dat

```diff
@@ -0,0 +1,22 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set gauge unitary
+set complex_mass_scheme False
+import model NMSSMHET_UFO --modelname
```

Comment:
Is your model by default 4f scheme or 5f scheme?

Reply:
When I import the model, the definition of the proton is 'p = g u c d s u~ c~ d~ s~'.

That would mean 4f scheme, right?

Reply:
yes